staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -44,6 +45,15 @@ const (
 // ServerRunOptions contains the options while running a generic api server.
 type ServerRunOptions struct {
 	AdvertiseAddress net.IP
+	// PeerCAFile is the ca bundle used by this kube-apiserver to verify peer apiservers'
```

Comment:
I expected this in kube-apiserver or kube-aggregator (alongside the proxy certificate options that enable using this), not here. Was that considered?